scenes/world_map/frays_end.tscn

```diff
@@ -269,14 +267,14 @@ sprite_frames = ExtResource("22_e453w")
 
 [node name="LoreQuestElder" parent="NPCs" node_paths=PackedStringArray("eternal_loom") instance=ExtResource("20_xt3dw")]
 position = Vector2(690, 600)
-quest_scene = ExtResource("11_0a1i7")
+quest_directory = "res://scenes/quests/lore_quests/"
```

Comment:
This doesn't work in the web build, for unknown reasons. I'll investigate tomorrow.

Reply:
Pushed a new patch that fixes this.